scalafmt-tests/src/test/resources/unit/Case.case

```diff
@@ -81,3 +81,15 @@ case q"""...${stats: Seq[Stat]} """ =>
 case  List(xs @ _*) => a
 >>>
 case List(xs @ _*) => a
+<<< #2258 Evenly align case values
+newlines.source = keep
```

Comment:
btw, other `source` options should also be tested.

Reply:
Refactored it without `source`